config/server.js

```diff
@@ -1,10 +1,13 @@
 module.exports = ({ env }) => ({
-  host: env('HOST', '0.0.0.0'),
-  port: env.int('PORT', 1337),
+  host: env("HOST", "0.0.0.0"),
+  port: env.int("PORT", 1337),
   admin: {
     auth: {
-      secret: env('ADMIN_JWT_SECRET', '255a8bb6c62d421ce19160f15bea8aa9'),
+      secret: env("ADMIN_JWT_SECRET", "255a8bb6c62d421ce19160f15bea8aa9"),
```

Comment:
Should these be configured with Azure Secrets(or similar, called secrets in aws)

Reply:
This takes the value by default from the env variable ADMIN_JWT_SECRET, which is defined as an Application setting in Azure Web App, and I think those are fine for this use case:

> Application settings are encrypted at rest and transmitted over an encrypted channel. You can choose to display them in plain text in your browser by using the controls below. Application Settings are exposed as environment variables for access by your application at runtime.